core/src/main/java/io/grpc/inprocess/InProcessChannelBuilder.java

```diff
@@ -24,6 +24,7 @@
 import io.grpc.internal.ConnectionClientTransport;
 import io.grpc.internal.GrpcUtil;
 import io.grpc.internal.SharedResourceHolder;
+
```

Comment:
We only put extra line between static and non-static imports. Remove here and elsewhere.